app/main.py

```diff
@@ -1,6 +1,16 @@
 from typing import Callable
 
 
-def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+def cache(func: Callable, cached_results: {} = None) -> Callable:
```

Comment:
Your decorator shouldn't be taking extra params other then a function. So you can simply declare this dict inside of decorator